### DISCUSSION TITLE
Looking to Contribute

### DISCUSSION THREAD

**Discussion by nadavsuissa** [networkx/networkx#5199] [Ideas]
Hello, we are a group of Cs students looking to contribute to an open source code program.
In this case we would be looking to contribute by Implementing so interesting Graph and Network algorithms

Would this be something that you might accept as a contribution if implement at a high enough level?
Would this be something that you are interested in?

**Comment by dschult** [comment]
Anyone can contribute to our project. 
See the file `CONTRIBUTING.rst` file in the github repository.
Enjoy! :}
